src/components/domain/auth/login/LoginModal.tsx

```diff
@@ -43,34 +44,28 @@ export default function LoginModal() {
   } = formMethods;
 
   const onSubmit = async (data: LoginForm) => {
-    try {
-      const response = await login(data);
-      const { accessToken, refreshToken } = response.data;
-
-      // 토큰을 로컬 스토리지에 저장
-      localStorage.setItem('accessToken', accessToken);
-      localStorage.setItem('refreshToken', refreshToken);
+    loginMutation.mutate(data);
+  };
 
-      closeModal();
-      alert('로그인에 성공했습니다.');
-    } catch (error) {
-      if (axios.isAxiosError(error)) {
-        const errorMessage = error.response?.data?.message || '로그인 중 오류가 발생했습니다.';
-        alert(errorMessage);
-      } else {
-        alert('로그인 중 오류가 발생했습니다.');
-      }
+  // 엔터 입력 시 로그인 폼 제출
+  const handleKeyDown = (event: React.KeyboardEvent) => {
```

Comment:
여기 타입 조금 더 명확하게 주면 좋을 것 같습니다
```suggestion
  const handleKeyDown = (event: React.KeyboardEvent<HTMLInputElement>) => {
```

Reply:
✅ f1aa2b3ea7f059ba8dbab821d3f15ccbe5d57510